### DISCUSSION TITLE
Default showClear behavior on DataSearch

### DISCUSSION THREAD

**Discussion by redlinecodes** [appbaseio/reactivesearch#2026] [Q&A]
If I enable showClear prop on DataSearch an "X" appears in the search form and I'm thinking it clears out the current query string. However, it does not. It still performs a search (or at least submits an empty string as the query).  Is there a way to change that behavior to simply clearing out the current query in the search form?

**Comment by SavvyShah** [comment]
It doesn't happen in the latest version(v3.42.1). Do you find the issue with [demo](https://codesandbox.io/s/github/appbaseio/reactivesearch/tree/next/packages/web/examples/SearchBox) in docs?
